enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+$ tree
+.
+├── 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0
+│   ├── VERSION
+│   ├── json
+│   └── layer.tar
+├── f6bbe84c78f5d0725d248207f92a23edfcbe66d3016f841018c03f933341dae3.json
+├── manifest.json
+└── mybundle.tar
+$ 
+``` 
+
+Now, see that all manifests are in the `layer.tar`:
+
+```
+$ tar -xvf 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0/layer.tar
+$ tree
+.
+├── 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0
+│   ├── VERSION
+│   ├── json
+│   └── layer.tar
+├── f6bbe84c78f5d0725d248207f92a23edfcbe66d3016f841018c03f933341dae3.json
+├── manifest.json
+├── manifests
+│   ├── myopertor.v0.5.5.clusterserviceversion.yaml
+│   ├── apps_v1alpha1_apimanager_crd.yaml
+│   ├── capabilities_v1alpha1_api_crd.yaml
+│   ├── capabilities_v1alpha1_binding_crd.yaml
+│   ├── capabilities_v1alpha1_limit_crd.yaml
+│   ├── capabilities_v1alpha1_mappingrule_crd.yaml
+│   ├── capabilities_v1alpha1_metric_crd.yaml
+│   ├── capabilities_v1alpha1_plan_crd.yaml
+│   └── capabilities_v1alpha1_tenant_crd.yaml
+├── metadata
+│   └── annotations.yaml
+├── mybundle.tar
+└── root
+    └── buildinfo
+        ├── Dockerfile-myoperator-rhel7-operator-metadata-2.8.2-5
+        └── content_manifests
+            └── myopertor-bundle-container-2.8.2-5.json
+```
+
+#### Using SDK bundle validate
+
+[SDK](https://github.com/operator-framework/operator-sdk) allows us to validate all bundles: 
+
+```
+ $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --output=json-alpha1
+
+INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
+{
+    "passed": false,
+    "outputs": [
+        {
+            "type": "error",
+            "message": "Error: Value : (memcached-operator.v0.0.1) csv.Spec.Icon elements should contain both data and mediatype"
+        }
+    ]
+}
+```
+
+Currently, the validators used by it will only ensure the bundle [spec definition](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md) and the standard requirements to publish an operator in the OperatorHub.io. For further information also check the proposal [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub].
+
+#### Using SDK scorecard bundle
+
+For the SDK tool be able to run scorecard tests its configuration requires to be wrote in the bundle directory (e.g `tests/scorecard/config.yaml` see [here](https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/tests/scorecard/config.yaml)). Operator bundles which are build with SDK will have the scorecard tests configured by default. (e.g see [here](https://github.com/operator-framework/community-operators/tree/master/community-operators/namespace-configuration-operator/1.0.1)).
+ 
```

Comment:
I'd guess that the CI/Pipeline people would make use of a custom scorecard image that implemented these audit tests.  So, I'd probably target this towards those users (e.g. CVP).  For that purpose, I'd construct a custom scorecard image that will execute these tests.  Then, CVP or others could modify their scorecard config's to include these tests if they want.  Users can either add the custom scorecard image configuration into their bundle's config.yaml directly (and therefore in their source code), or CVP or others can run scorecard with the --config flag to run scorecard with an externalized config.yaml which would then include this sort of custom image test configuration.  If this custom image proves useful, then you could determine later on to add this into the SDK generated config.yaml at a later point.

Reply:
I am only saying here that: 
- audit tool requires to write the config when/only it does not exist
- in oder to avoid it write a config that is not compatible with SDK version X used to run the checks we would need a command that does this job

**IMPORTANT** It is a need that no longer exists by just adding audit in SDK. +1 reason here for in POV we move forward with the solution shipped in SDK.

Reply:
It is a great idea. Addressed!